### DISCUSSION TITLE
Twitter logo at front page

### DISCUSSION THREAD

**Discussion by Ayush9026** [Chia-Network/chia-blockchain#16573] [Ideas]
### What happened?

Twitter logo at front page is old marked  below in picture replace  logo 'x' .
![Chia-network front page](https://github.com/Chia-Network/chia-blockchain/assets/114745463/2cb9916c-310b-463b-af5d-39f8a17aaa86)

### Version

current

### What platform are you using?

Windows

### What ui mode are you using?

GUI

**Comment by BrandtH22** [comment]
@Ayush9026 thank you for the info, I have passed this along to the appropriate teams.

I am going to migrate this ticket to a discussion so we can keep track of it :+1:
